### PR TITLE
Add tests for transferable streams

### PR DESCRIPTION
**Pull request by ricea**
The Streams Standard change https://github.com/whatwg/streams/pull/1053
adds the ability to transfer streams to a different realm using
postMessage(). Add tests for this feature.

### CODE REVIEW

File: streams/transferable/readable-stream.html [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+      controller.error('foo');
+    }
+  });
+  const reader = rs.getReader();
+  let threw = false;
+  try {
+    await reader.read();
+  } catch (e) {
+    threw = true;
+    assert_equals(e, 'foo', 'error should be passed through');
```
[the remark this answers is not in the document]
Comment (ricea) [reply streams/transferable/readable-stream.html]: Done, thanks.

**Comment by ricea** [comment]
@domenic Could you take a look at this?

File: streams/transferable/resources/helpers.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+// Create a ReadableStream that will pass the tests in
```
Comment (domenic) [inline streams/transferable/resources/helpers.js:3]: Nit: in other helper files I've preferred to use `self.X = ...` to make it clearer what is exported. Possibly even hiding non-exported values inside a closure, like https://github.com/web-platform-tests/wpt/blob/master/streams/resources/rs-utils.js does.

**Review by domenic — APPROVED** [review]
LGTM!

File: streams/transferable/resources/helpers.js (hunk 2 above)
Comment (ricea) [reply streams/transferable/resources/helpers.js:3]: I will fix this in a follow-up change.

**Pull request merged**